ui/components/app/alerts/unconnected-account-alert/unconnected-account-alert.js

```diff
@@ -63,7 +63,7 @@ const UnconnectedAccountAlert = () => {
             className="unconnected-account-alert__checkbox-label"
             htmlFor="unconnectedAccount_dontShowThisAgain"
           >
-            {t('dontShowThisAgain')}
+            <Box>{t('dontShowThisAgain')}</Box>
```

Comment:
This should be using Text or Label component, I think

Reply:
Good shout.  Updated!